pom.xml

```diff
@@ -129,71 +121,77 @@
     <failsafe.forkCount>1</failsafe.forkCount>
     <failsafe.groups />
     <failsafe.reuseForks>false</failsafe.reuseForks>
-    <hadoop.version>3.3.5</hadoop.version>
     <!-- prevent introduction of new compiler warnings -->
     <maven.compiler.failOnWarning>true</maven.compiler.failOnWarning>
     <maven.compiler.release>11</maven.compiler.release>
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <maven.javadoc.failOnWarnings>true</maven.javadoc.failOnWarnings>
     <maven.site.deploy.skip>true</maven.site.deploy.skip>
+    <maven.site.skip>true</maven.site.skip>
     <!-- surefire/failsafe plugin option -->
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
+    <!-- versions-maven-plugin ignore patterns for snapshots, alpha, beta, milestones, and release candidates -->
+    <maven.version.ignore>.+-SNAPSHOT,(?i).*(alpha|beta)[0-9]*,(?i).*[.-](m|rc)[0-9]+</maven.version.ignore>
     <minimalJavaBuildVersion>11</minimalJavaBuildVersion>
-    <minimalMavenBuildVersion>3.5.0</minimalMavenBuildVersion>
-    <powermock.version>2.0.9</powermock.version>
+    <minimalMavenBuildVersion>3.6.0</minimalMavenBuildVersion>
     <!-- timestamp for reproducible outputs, updated on release by the release plugin -->
     <project.build.outputTimestamp>2023-06-14T05:41:08Z</project.build.outputTimestamp>
     <rat.consoleOutput>true</rat.consoleOutput>
-    <slf4j.version>2.0.7</slf4j.version>
     <sourceReleaseAssemblyDescriptor>source-release-tar</sourceReleaseAssemblyDescriptor>
     <surefire.excludedGroups />
     <surefire.failIfNoSpecifiedTests>false</surefire.failIfNoSpecifiedTests>
     <surefire.forkCount>1C</surefire.forkCount>
     <surefire.groups />
     <surefire.reuseForks>true</surefire.reuseForks>
-    <surefire.version>3.1.0</surefire.version>
-    <!-- Thrift version -->
-    <thrift.version>0.17.0</thrift.version>
     <unitTestMemSize>-Xmx1G</unitTestMemSize>
-    <!-- ZooKeeper version -->
-    <zookeeper.version>3.8.1</zookeeper.version>
+    <!-- dependency and plugin versions managed with properties -->
+    <version.auto-service>1.1.1</version.auto-service>
+    <version.bouncycastle>1.70</version.bouncycastle>
+    <version.curator>5.5.0</version.curator>
+    <version.errorprone>2.20.0</version.errorprone>
+    <version.hadoop>3.3.6</version.hadoop>
+    <version.opentelemetry>1.27.0</version.opentelemetry>
+    <version.powermock>2.0.9</version.powermock>
+    <version.slf4j>2.0.7</version.slf4j>
+    <version.thrift>0.17.0</version.thrift>
+    <version.zookeeper>3.8.1</version.zookeeper>
   </properties>
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>com.fasterxml.jackson</groupId>
         <artifactId>jackson-bom</artifactId>
-        <version>2.15.1</version>
+        <version>2.15.2</version>
```

Comment:
Maybe. But, I was keeping it limited for now. Adding new properties gets excessive fast. Also, some of these only exist for dependency convergence of transitive deps, not because we actually directly depend on them. Seems weird to add new properties for those.

If we want to add any new properties for these, we can do that in a subsequent change.